pkg/guestagent/hwclock/hwclock_linux.go

```diff
@@ -0,0 +1,23 @@
+package hwclock
+
+import (
+	"bytes"
+	"os/exec"
+	"strings"
+	"time"
+)
+
+func Get() (t time.Time, err error) {
+	cmd := exec.Command("hwclock")
```

Comment:
Can we directly call ioctl?

https://man7.org/linux/man-pages/man4/rtc.4.html

Reply:
Good point. I will give it a try.

Reply:
@mikluko Do you need any help? I'm interested in getting this merged within the next week or two, so if there is anything I can do to help this move forward, please let me know!